Microsoft.Toolkit.Win32.UI.Controls/WinForms/WebView/WebView.Init.cs

```diff
@@ -0,0 +1,197 @@
+            Process = _webViewControl.Process;
+
+            EnsureInitialized();
+        }
+
+        private bool Initialized => _initializationState == InitializationState.IsInitialized;
+
+        private bool Initializing => _initializationState == InitializationState.IsInitializing;
+
+        private bool WebViewControlInitialized => _webViewControl != null;
+
+        void ISupportInitialize.BeginInit()
+        {
+            if (Initialized)
+            {
+                // TODO: Message
+                // Cannot initialize WebView since it is already completely initialized
+                throw new InvalidOperationException();
+            }
+
+            if (Initializing)
+            {
+                // TODO: Message
+                // Cannot initialize WebView since it is already being initialized
+                throw new InvalidOperationException();
+            }
+
+            _initializationState = InitializationState.IsInitializing;
+        }
+
+        void ISupportInitialize.EndInit()
+        {
+            if (!Initializing)
+            {
+                // TODO: Message
+                // Cannot complete WebView initialization that is not being initialized
+                throw new InvalidOperationException();
+            }
+
+            Initialize();
+        }
+
+        private void CheckInitialized()
+        {
+            if (!Initialized)
+            {
+                // TODO: Message
+                // Initialization incomplete
+                throw new InvalidOperationException();
+            }
+        }
+
+        // Ensures this class is initialized. Initialization involves using ISupportInitialize methods
+        private void EnsureInitialized()
+        {
+            // If already disposed, do nothing
+            if (IsDisposed)
+            {
+                throw new ObjectDisposedException(GetType().Name);
+            }
+
+            // If not already initialized and not already initializing
+            if (!Initialized && !Initializing)
+            {
+                ((ISupportInitialize)this).BeginInit();
+                ((ISupportInitialize)this).EndInit();
+            }
+        }
+
+        private void Initialize()
+        {
+            Verify.AreEqual(_initializationState, InitializationState.IsInitializing);
+            Verify.IsFalse(DesignMode);
+
+            // This is causing freezing
+            if (!DesignMode)
+            {
+                OSVersionHelper.ThrowIfBeforeWindows10RS4();
+
+                if (!WebViewControlInitialized)
+                {
+                    Verify.IsNull(Process);
+
+                    // Was not injected via ctor, create using defaults
+                    Process = new WebViewControlProcess(new WebViewControlProcessOptions
+                    {
+                        // TODO: Set secure default
+                        PrivateNetworkClientServerCapability = WebViewControlProcessCapabilityState.Enabled
```

Comment:
Update forthcoming allowing this state to be set in the designer and/or without utilizing a custom process object